R/build-quarto-articles.R

```diff
@@ -96,14 +108,19 @@ quarto_render <- function(pkg, path, quiet = TRUE, frame = caller_env()) {
   write_yaml(quarto_format(pkg), metadata_path)
 
   output_dir <- withr::local_tempdir("pkgdown-quarto-", .local_envir = frame)
-  quarto::quarto_render(
-    path,
-    metadata_file = metadata_path,
-    execute_dir = output_dir,
-    quarto_args = c("--output-dir", output_dir),
-    quiet = quiet,
-    as_job = FALSE
-  )
+
+  # set the lib path to the temporary pkgdown installation directory
+  lib_path <- .libPaths()[1]
+  withr::with_libpaths(lib_path, code = {
+    quarto::quarto_render(
+      path,
+      metadata_file = metadata_path,
+      execute_dir = output_dir,
+      quarto_args = c("--output-dir", output_dir),
+      quiet = quiet,
+      as_job = FALSE
+    )
+  })
```

Comment:
These are the relevant changes, air reformatted the remainder.

Reply:
`local_libpaths()` would be a bit simpler here. If you want to use air (which is fine), do you mind reformatting all of pkgdown in one PR and then using `usethis::use_air()` (from dev version) to ensure everyone in positron gets it automatically going forward?